package.json

```diff
@@ -21,7 +26,7 @@
     "update:patch": "npm-check-updates --configFileName .ncurc.patch.js --deep --dep dev,prod --target patch --upgrade && npm install"
   },
   "dependencies": {
```

Comment:
Why did you update only this one package? All but the `react` package of the `@amsterdam` dependencies can be removed if we update to 0.9.0 immediately. The dependency on `@babel\runtime` can then be removed as well.